core/src/main/kotlin/ch/ergon/dope/resolvable/clause/ISelectClause.kt

```diff
@@ -46,9 +45,10 @@ interface ISelectOrderByClause<T : ValidType> : ISelectLimitClause<T> {
 }
 
 interface ISelectGroupByClause<T : ValidType> : ISelectOrderByClause<T> {
-    fun orderBy(stringField: Field<StringType>) = SelectOrderByClause(stringField, this)
-    fun orderBy(stringField: Field<StringType>, orderByType: OrderByType) =
-        SelectOrderByTypeClause(stringField, orderByType, this)
+    fun orderBy(typeExpression: TypeExpression<out ValidType>, orderByType: OrderByType? = null) = SelectOrderByClause(
+        listOf(OrderExpression(typeExpression, orderByType)),
```

Comment:
see comment in SelectOrderByClause, if you use vararg you don't have to wrap the single type expression into a list

Reply:
done